Source/WebCore/rendering/CSSFilter.cpp

```diff
@@ -75,6 +75,11 @@ CSSFilter::CSSFilter(Vector<Ref<FilterFunction>>&& functions)
 {
 }
 
+static bool isIdentityBlurEffect(const BlurFilterOperation& blurOperation)
```

Comment:
You could either rely on C++ overloading and have all these static functions be called `isIdentityEffect()`, or you could add a `virtual bool isIdentity(std::optional<float> noopAmount) const` to the base class.